core/src/main/java/com/ritense/valtimo/autoconfigure/HttpSecurityAutoConfiguration.java

```diff
@@ -133,6 +135,13 @@ public UserHttpSecurityConfigurer userHttpSecurityConfigurer() {
         return new UserHttpSecurityConfigurer();
     }
 
+    @Order(315)
+    @Bean
+    @ConditionalOnMissingBean(GlobalHttpSecurityConfigurer.class)
+    public GlobalHttpSecurityConfigurer globalHttpSecurityConfigurer() {
```

Comment:
`GlobalHttpSecurityConfigurer` is a bit generic. Maybe rename to `GlobalSettingsHttpSecurityConfigurer`